Engine/Main/Globals.h

```diff
@@ -76,8 +76,8 @@ enum class Fonts
 # define SAVED_SCENE_PATH "Assets/Scenes/scene.scene"
 # define TMP_SCENE_PATH "Assets/Scenes/tmp_scene.scene"
 
-# define OPENGL_MAJOR_VERSION 4
-# define OPENGL_MINOR_VERSION 0
+# define OPENGL_MAJOR_VERSION 3
+# define OPENGL_MINOR_VERSION 3
```

Comment:
How this affect us?

Reply:
oh I changed this because I was working on a Mac and it doesn't support OPENGL 4. I forgot to change it back :)